js/bookmarker.js

```diff
@@ -4,18 +4,27 @@
  * Edge needs this to be a content_script that is injected, however.
```

Comment:
Is this line true anymore?

Reply:
It is, but also for every other browser, with this change.